_includes/header.html

```diff
@@ -17,7 +17,12 @@
 		</div>
 		<p class="headerName">{{ site.app_name }}</p>
 	</div>
-	<nav class="scroll notMobile">
+	<button class="menu-button" id="header-menu-btn">
+		<div></div>
+		<div></div>
+		<div></div>
```

Comment:
I had issues that sometimes the menu wouldn't open. I think there might be a bug with where if you click exactly on one of the divs it doesn't register it as a click. Could you look into that?

Reply:
Sure, I'll do when I'm available.

Reply:
I just disabled it (if someone ends up here from a websearch: Site configuration -> Build & deploy -> Continuous deployment -> Collaboration tools -> Netlify Drawer: Disable)

I also noticed just now (oops, should've tested that first before commenting) that it works fine on the LineageOS default browser, which is based on Chromium (org.lineageos.jelly) but it doesn't work on Firefox (well, I use [Fennec F-Droid](https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/) but that should be the same). Wonder why :(

Reply:
The main issue has been fixed, but I just found a bug on webkit engine that prevent me to do blur effect for the theme properly. So I'm going to fix a for a bit. After done, I'll ping you to do review again.